### PR TITLE
Hide menu button untill user is authenticated

### PR DESCRIPTION
**Pull request by Trulsaa**

### CODE REVIEW

File: packages/frontend/src/App.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
             </router-link>
           </div>
           <div class="md-toolbar-section-end">
-            <hamburger />
+            <hamburger v-if="userFound" />
```
Comment (boye-s) [inline packages/frontend/src/App.vue:18]: Just semantics, but I would probably move the `v-if` to the encapsulating `<div>` as it serves no purpose if all its content is hidden anyway.

**Review by boye-s — APPROVED** [review]
I left a comment, but just minor semantics. 🥇

File: packages/frontend/src/App.vue (hunk 1 above)
Comment (Trulsaa) [reply packages/frontend/src/App.vue:18]: The class on the div has impact on the placement of other divs in the header.

Comment (boye-s) [reply packages/frontend/src/App.vue:18]: That is a fair point :)

**Pull request merged**